src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateAction.java

```diff
@@ -0,0 +1,39 @@
+package org.csanchez.jenkins.plugins.kubernetes;
+
+import hudson.Extension;
+import hudson.model.Action;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import jenkins.model.TransientActionFactory;
+
+@Extension
+public class PodTemplateAction extends TransientActionFactory<KubernetesCloud> implements Action{
+    
+    @Override
+    public String getIconFileName() {
+        return "symbol-copy-outline plugin-ionicons-api";
```

Comment:
It seems you need to have the `plugin-ionicons-api` plugin installed but I did not had to add more deps.

Reply:
Currently comes as a transitive dep of `cloudbees-folder`. You should add it if you depend on it explicitly.

```
+- org.jenkins-ci.plugins:cloudbees-folder:jar:6.848.ve3b_fd7839a_81:compile
|  \- io.jenkins.plugins:ionicons-api:jar:56.v1b_1c8c49374e:compile
```

Reply:
The real question is whether we want to add a dependency for one icon.